apigee_edge.module

```diff
@@ -992,11 +993,21 @@ function template_preprocess_app_credential(array &$variables) {
       ];
     }
     elseif ($def['value_type'] === 'status') {
-      $variables['primary_wrapper'][$property]['value'] = [
-        '#type' => 'status_property',
-        '#value' => $value,
-        '#indicator_status' => $value === AppCredentialInterface::STATUS_APPROVED ? StatusPropertyElement::INDICATOR_STATUS_OK : StatusPropertyElement::INDICATOR_STATUS_ERROR,
-      ];
+      $expires = $credential->getExpiresAt();
+      if ($expires && $expires->getTimestamp() < \Drupal::time()->getRequestTime()) {
+        $variables['primary_wrapper'][$property]['value'] = [
```

Comment:
DRY. The render array is always the same, just the `#value` and the `#indicator_status` has to be calculated individually.